Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:buster-v1.4.0
```

Comment:
We can switch to using distroless image, but we would need to set `runAsUser: 0` as the plugin needs access to azure.json for creds.

Reply:
discussed as a follow-up

Reply:
https://github.com/Azure/kubernetes-kms/issues/83